setup.py

```diff
@@ -5,7 +5,7 @@
 from setuptools import find_packages, setup
 
 setup(
-    name="celery_sqlalchemy_scheduler",
+    name="uxi_celery_scheduler",
```

Comment:
We usually get away this dashes.
```suggestion
    name="uxi-celery-scheduler",
```